File: Cellar.Hub.Admin/SPA/app/entities/CellarPlace.ts

export class CellarPlace {

    //Alternative for constructor
    New(json: any): CellarPlace {
        this.id = json.id;
        this.name = json.name;
        this.state = json.state;
        this.path = json.path;

        this.country = json.country;
        this.city = json.city;
        this.street = json.city;
        this.zipcode = json.city;
        this.latitude = json.city;
        this.longtitude = json.city;

        return this
    }

    public id: string;

    public name: string;

    public state: string;
    public path: string = "/";

    // Country, City, Street
    public country: string;
    public city: string;
    public street: string;
    public zipcode: string;

    // GPS - latitude, longtitude
    public latitude: string;
    public longtitude: string;

}

File: Cellar.Hub.Api/Models/CellarPlace.cs
using System.Collections.Generic;
using System.Security.Claims;
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson.Serialization.IdGenerators;

namespace Cellar.Hub.Api.Models
{
    //OBJECT FOR PLACE STRUCTURE (solve World problem)
    public class CellarPlace
    {

        [BsonId(IdGenerator = typeof(StringObjectIdGenerator))]
        [BsonElement("id")]
        public string Id { get; set; }  

        [BsonElement("name")]
        public string Name { get; set; }

        [BsonElement("state")]
        public string State { get; set; }

        [BsonElement("path")]
        public string Path { get; set; }

        // Country, City, Street
        [BsonElement("country")]
        public string Country {get;set;}
        [BsonElement("city")]
        public string City {get;set;}
        [BsonElement("street")]
        public string Street{get;set;}
        [BsonElement("zipcode")]
        public string Zipcode{get;set;}

        // GPS - latitude, longtitude
        [BsonElement("latitude")]
        public string Latitude {get;set;}
        [BsonElement("longtitude")]
        public string Longtitude {get;set;}



    }
}